test/verify/check-networkmanager-checkpoints

```diff
@@ -161,7 +161,7 @@ class TestNetworkingCheckpoints(NetworkCase):
         # back.
 
         def checkpoint_was_destroyed():
-            lines = m.execute("journalctl -n 1000 -u NetworkManager | grep op=").split("\n")
+            lines = m.execute("journalctl -u NetworkManager | grep op=").split("\n")
```

Comment:
A few days ago I learned that `journalctl` has a `-g` option so we could drop `| grep op=` in the future.